src/Altinn.Platform/Altinn.Platform.Storage/Storage/Altinn.Platform.Storage.csproj

```diff
@@ -12,11 +12,11 @@
   <ItemGroup>
     <PackageReference Include="Altinn.Common.PEP" Version="1.0.16-alpha" />
```

Comment:
should we update PEP now that wer're in this project anyways?

Reply:
Done, did.